pkg/types/hashedrekord/hashedrekord.go

```diff
@@ -59,21 +56,16 @@ func ToLogEntry(hr *pb.HashedRekordRequestV0_0_2) (*pb.HashedRekordLogEntryV0_0_
 	if err != nil {
 		return nil, fmt.Errorf("parsing public key: %w", err)
 	}
-	var alg crypto.Hash
-	switch hr.Data.Algorithm {
-	case v1.HashAlgorithm_SHA2_384:
-		alg = crypto.SHA384
-	case v1.HashAlgorithm_SHA2_512:
-		alg = crypto.SHA512
-	default:
-		alg = crypto.SHA256
-	}
-	if err := sigObj.Verify(nil, keyObj, options.WithDigest(hr.Data.Digest), options.WithCryptoSignerOpts(alg)); err != nil {
+
+	// TODO: Look up hash with sigstore/sigstore's GetAlgorithmDetails(hr.Signature.Verifier.KeyDetails).GetHashType()
```

Comment:
Needed to undo this temporarily, we'll add this back in https://github.com/sigstore/rekor-tiles/pull/154 when it uses the AlgorithmRegistry.

Reply:
This breaks #154 because now the algorithm is hardcoded.